janusgraph-hadoop/src/main/java/org/janusgraph/hadoop/formats/util/JanusGraphVertexDeserializer.java

```diff
@@ -59,14 +60,14 @@ public JanusGraphVertexDeserializer(final JanusGraphHadoopSetup setup) {
         this.idManager = setup.getIDManager();
     }
 
+    private Boolean edgeExists(Vertex vertex, RelationCache possibleDuplicate) {
```

Comment:
Not need to return wrapper type. I would simply return `boolean` instead of `Boolean` here.

Reply:
I originally left it this way because the previous implemnetation also returned the wrapper type. However, I agree with you and will change it.